src/molecules/TogglePanel.tsx

```diff
@@ -0,0 +1,21 @@
+import { TogglePanelContainer, CircleIcon, ToggleItemContainer } from '@/atoms/TogglePanelAtoms';
+
+export const TogglePanel = ({ data, toggleState, toggleHandler }: TogglePanel) => {
```

Comment:
Currently toggle panel only assumes it will be fed geojson data; additional support would need to be codefied (in this component) to handle other data types (like a sole X,Y coordinate representing an indicator)

Reply:
I this this is a fair standard format for now